src/useCases/postFeedback.ts

```diff
@@ -0,0 +1,29 @@
+import config from "lib/config"
+import getEmailer from "lib/getEmailer"
+import PromiseResult from "types/PromiseResult"
+import logger from "utils/logger"
+
+const postFeedback = (feedback: string, currentUserEmail: string): PromiseResult<void> => {
+  const sendFeedbackTo = "kayleigh.derricutt@madetech.cjsm.net" // sorry Kayleigh :(
+
+  const emailer = getEmailer(sendFeedbackTo)
+  const emailContent = {
+    subject: "New Feedback",
+    html: "",
+    text: `User ${currentUserEmail} has written the following feedback: '${feedback}'`
```

Comment:
`currentUserEmail` might be set to `""` here - which would maybe be a little confusing.

As mentioned above, perhaps we should make the `currentUserEmail` parameter optional, and then check it here to determine the text?